CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 
 ## unreleased
 * Breaking Changes
+  * Card 
+    * Update `Attributes` field, adding `customer` and `vault` properties
+      `customer` is of optional `Customer` type, containing property `id` of String type
+      `vault` is of optional `Vault` type, containing `storeInVault` field of `StoreInVault` enum type
+       with one value, `.onSuccess`
```

Comment:
The `Attributes` object is internal correct? If so we do not need a changelog entry for it.